var/spack/repos/builtin/packages/py-bluepymm/package.py

```diff
@@ -26,6 +26,8 @@ class PyBluepymm(PythonPackage):
     depends_on('py-lxml', type='run')
     depends_on('py-sh', type='run')
     depends_on('neuron', type='run')
+    depends_on('py-h5py@:2.99', type='run')
+    depends_on('py-pyyaml', type='run')
 
     def setup_run_environment(self, env):
         env.unset('PMI_RANK')
```

Comment:
Thanks for the update. The h5py version can also be >=3.0 for bluepymm.

Reply:
Thanks, I reverted the last commit.